frontend/web/components/GithubStar.tsx

```diff
@@ -21,28 +21,41 @@ const GithubStar: FC<GithubStarType> = ({}) => {
         .then(function (res) {
           setStars(res.stargazers_count)
         })
+        .catch(() => {})
     }
   }, [planName])
 
   if (planName === planNames.enterprise) {
     return <></>
   }
 
-  return (
-    <>
+  if (!stars) {
+    return (
       <a
         style={{ width: 90 }}
         target='_blank'
         href='https://github.com/flagsmith/flagsmith'
         className='btn btn-sm btn-with-icon text-body'
         rel='noreferrer'
       >
-        <div className='d-flex flex-row justify-content-center align-items-center'>
-          <IonIcon style={{ fontSize: 16 }} icon={logoGithub} />
-          <div className='ms-1'>{stars}</div>
-        </div>
+        <IonIcon style={{ fontSize: 16 }} icon={logoGithub} />
       </a>
-    </>
+    )
+  }
+
+  return (
+    <a
+      style={{ width: 90 }}
+      target='_blank'
+      href='https://github.com/flagsmith/flagsmith'
+      className='btn btn-sm btn-with-icon text-body'
+      rel='noreferrer'
+    >
+      <div className='d-flex flex-row justify-content-center align-items-center'>
+        <IonIcon style={{ fontSize: 16 }} icon={logoGithub} />
+        <div className='ms-1'>{stars}</div>
+      </div>
+    </a>
```

Comment:
Can we make this more DRY? I'm sure it's more complicated than this, but would something _like_ the following work? 

```javascript
  return (
    <a
      style={{ width: 90 }}
      target='_blank'
      href='https://github.com/flagsmith/flagsmith'
      className='btn btn-sm btn-with-icon text-body'
      rel='noreferrer'
    >
      <div className='d-flex flex-row justify-content-center align-items-center'>
        <IonIcon style={{ fontSize: 16 }} icon={logoGithub} />
        { !stars ? <div className='ms-1'>{stars}</div> : '' }
      </div>
    </a>
  )
```

Reply:
Normally I prefer the early return because it's easier to read but in this case I agree we can make it less repetitive and smaller